DASCharter-2020.html

```diff
@@ -1,677 +1,1357 @@
+            <a href="https://www.w3.org/TR/orientation-sensor/">Orientation
+            Sensor</a>
+          </dt>
+          <dd>
+            <p>
+              An API to monitor the device’s physical orientation in relation
+              to a stationary three dimensional Cartesian coordinate system
+            </p>
+            <p class="draft-status">
+              <b>Draft state:</b> Candidate Recommendation
+            </p>
+            <p>
+              <b>Adopted Candidate Recommendation</b>: TBD
+            </p>
+            <p>
+              <b>Reference Candidate Recommendation</b>: TBD
+            </p>
+            <p>
+              Produced under <b>Working Group Charter:</b> TBD
+            </p>
+          </dd>
+          <dt>
+            <a href=
+            "https://github.com/SamsungInternet/Explainers/blob/master/Foldables/FoldState.md#angle-sensor-based-on-genericsensor-api">
+            Fold Angle Sensor</a>
+          </dt>
+          <dd>
+            <p>
+              An API to monitor the device’s fold angle
+            </p>
+            <p class="draft-status">
+              <b>Draft state:</b> Explainer
+            </p>
+            <p>
+              <b>Adopted Candidate Recommendation</b>: TBD
+            </p>
+            <p>
+              <b>Reference Candidate Recommendation</b>: TBD
+            </p>
+            <p>
+              Produced under <b>Working Group Charter:</b> TBD
+            </p>
+          </dd>
+          <dt>
+            <a href=
+            "https://www.w3.org/TR/orientation-event/">DeviceOrientation Event
+            specification</a>
+          </dt>
+          <dd>
+            <p>
+              An event-based API that provides information about the physical
+              orientation and motion of a hosting device
+            </p>
+            <p>
+              <b>Draft state</b>: Working Draft
+            </p>
+            <p>
+              <b>Adopted Working Draft</b>: TBD
+            </p>
+            <p>
+              <b>Reference Draft</b>: TBD
+            </p>
+            <p>
+              Produced under <b>Working Group Charter:</b> TBD
+            </p>
+          </dd>
+          <dt>
+            <a href="https://www.w3.org/TR/geolocation-sensor/">Geolocation
+            Sensor</a>
+          </dt>
+          <dd>
+            <p>
+              An API for obtaining geolocation reading from the hosting device,
+              based on the Generic Sensor API
+            </p>
+            <p>
+              <b>Draft state</b>: Working Draft
+            </p>
+            <p>
+              <b>Adopted Working Draft</b>: TBD
+            </p>
+            <p>
+              <b>Reference Draft</b>: TBD
+            </p>
+            <p>
+              Produced under <b>Working Group Charter:</b> TBD
+            </p>
+          </dd>
+          <dt>
+            <a href="https://wicg.github.io/netinfo/">Network Information
+            API</a>
```

Comment:
Given the discussion on #78 do we want to drop this for now?

Reply:
Since this API is in scope of the current charter already with the same WICG provision, I’d lean on keeping this option since adding new work to a WG is costly business. We can add Web Perf WG explicitly as another possible home.